examples/django/oauth_app/slack_listeners.py

```diff
@@ -22,14 +22,16 @@
 
 app = App(
     signing_secret=signing_secret,
-    installation_store=DjangoInstallationStore(
```

Comment:
If you pass installation_store this way, it does not work with the token rotation. I will create a new issue for this.

Reply:
https://github.com/slackapi/bolt-python/issues/409